docs/source/Quickstart/04_non-virt.md

```diff
@@ -10,6 +10,11 @@ In order to do so, some steps should be done beforehand.
 
 ### 1. Get the project and its dependencies
 ```Bash
+$ cd $HOME
+$ mkdir -p $HOME/src/github.com/openstack-k8s-operators/
```

Comment:
Why is this needed? Those are paths that we typically use in CI, but the shouldn't be necessary. Agree in the add an explicit `cd $HOME`

Reply:
Looks like the example config does not overrides the location everywhere.
ATM better to copy the ci paths, until a fixing change lands at least the doc sample would be WORKSFORME.

Reply:
Maybe the without ".com" version should be the default .
checking ..